packages/core/admin/admin/src/components/GuidedTour/Stepper/Modal/StepperModal.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { pxToRem } from '@strapi/helper-plugin';
+import { Typography } from '@strapi/design-system/Typography';
+import { Button } from '@strapi/design-system/Button';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import Content from '../../GuidedTourModal/Content';
+import StepLine from '../StepLine';
+import StepNumberWithPadding from './StepNumberWithPadding';
+
+const LeftSideBox = styled(Flex)`
+  margin-right: ${({ theme }) => theme.spaces[8]};
```

Comment:
marginRight is a prop of Flex (Box) you can pass `marginRight={8}` to LeftSideBox